### DISCUSSION TITLE
Questions about LHC free version, self hosted.

### DISCUSSION THREAD

**Discussion by wahmed84** [LiveHelperChat/livehelperchat#1969] [Q&A] [answered]
First of all thanks for this great tool.
Few questions:
Can anyone please tell me if the free version that i downloaded and hosted on my own server is free for life or it has some limitations ? because it also says "14 days evaluation". 
Is it going to stop after 14 days of evaluation?

**Comment by remdex** [comment, marked as answer]
It's always free if you host it. There is no limitation...

**Reply by wahmed84** [reply]
Hi remdex,

Great, Thanks for your quick response.

**Reply by wahmed84** [reply]
Also, I am facing an issue.. Don't know if this is right place to ask.
I configured live helper chat on apache port 80 which is under load balancer with https.
Now when i embed the script on my website i get mixed content error, i tried live helper chat url with http, https, with only //... but in all case the js script is trying to load with http, can you help?

**Reply by remdex** [reply]
Make sure you embed code using https protocol. Https you can choose in embed code generation.
Also read this. You can set explicitly to use only https...
https://doc.livehelperchat.com/docs/development/http-and-https

**Reply by wahmed84** [reply]
Thanks, i resolved this issue.
